examples/Hello_World/hello_world.py

```diff
@@ -11,7 +11,7 @@
 
 # Create a log file.
 logger = logging.getLogger()
-logger.addHandler(logging.StreamHandler(file("hello_world.log", "w")))
+logger.addHandler(logging.FileHandler("hello_world.log", "w", encoding="utf8"))
```

Comment:
nit: "u8"? "utf-8"? "utf8"? 😂

Reply:
sigh. looks like the rest of the codebase uses `utf-8`. fixing.

Reply:
`utf-8` is the official name of the Python codec (see the exception message below for an example); the other variants are aliases. (UTF-8 is the official Unicode spelling.)
```
>>> b'\x80abc'.decode('utf8')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 0: invalid start byte
```